nemo/collections/llm/gpt/data/hf_dataset.py

```diff
@@ -61,7 +61,10 @@ def pad_within_micro(batch, pad_token_id):
             max_len = max(map(len, batch))
             return [item + [pad_token_id] * (max_len - len(item)) for item in batch]
 
-        keys = list(filter(lambda x: x in batch[0], ['tokens', 'labels', 'position_ids', 'loss_mask']))
+        print(batch[0])
+        keys = list(
+            filter(lambda x: x in batch[0], ['input_features', 'tokens', 'labels', 'position_ids', 'loss_mask'])
```

Comment:
any way to avoid this? this change will break LLM peft

Reply:
Yeah, that was mistakenly pushed. Reverted already.